spec/test_apps/rails-5.1.4/app/assets/javascripts/application.js

```diff
@@ -0,0 +1 @@
+/Users/zubin/src/envato/guide/spec/test_apps/shared/app/assets/javascripts/application.js
```

Comment:
what are these files and why do they have your filepath in them?

Reply:
@shevaun These are symlinked from `spec/test_apps/shared/app`. 

I intended to use relative symlinks but apparently did not.